README.md

```diff
@@ -15,53 +15,81 @@ yarn add --dev storybook-addon-apollo-client
 npm install -D storybook-addon-apollo-client
 ```
 
-## As a decorator in a story
+**babel**
+
+You must have `babel-plugin-graphql-tag` installed due to this issue: https://github.com/graphql/graphql-js/pull/2864
```

Comment:
🔥 Awesome that you were able to track down the 'source' of this and put up a fix!